components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -400,7 +400,7 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         val (nioBuf, len) = capabilitiesToBuffer(supportedCapabilities)
         rustCall { e ->
             val ptr = Native.getDirectBufferPointer(nioBuf)
-            LibFxAFFI.INSTANCE.fxa_initialize_device(this.handle.get(), name, deviceType.toNumber(), ptr, len, e)
+            LibFxAFFI.INSTANCE.fxa_initialize_device(this.handle.get(), name, deviceType.ordinal, ptr, len, e)
```

Comment:
I believe this is the wrong fix, we should keep (and fix) `toNumber` instead using these values: https://github.com/mozilla/application-services/blob/77bd14de63df6b3d974e97dd069fb94e6929526f/components/fxa-client/src/fxa_msg_types.proto#L41-L45 (a comment to this line wouldn't hurt either).

Reply:
Sounds good!